src/osmf/js/osmf_flash.js

```diff
@@ -2,6 +2,11 @@
  * OSMF flash video plugin
  */
 
+require("../../../html5-common/js/utils/InitModules/InitOO.js");
+require("../../../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+require("../../../html5-common/js/utils/InitModules/InitOOHazmat.js");
+require("../../../html5-common/js/utils/constants.js");
+require("../../../html5-common/js/utils/environment.js");
```

Comment:
You may not need environment in your file; i don't see these used anywhere.  Also I saw that you added a bind to this down below.  Does the document readystate return the wrong value when not bound?  I'm asking because if you don't need to use underscore then you also won't need hazmat and underscore libraries.   Both are available for use so feel free, just wondering